app/assets/stylesheets/application.css

```diff
@@ -4,6 +4,7 @@
  * the top of the compiled file, but it's generally better to create a new file per style scope.
  *
  *= require c3
+ *= require patternfly-bootstrap-treeview/dist/bootstrap-treeview.min.css
```

Comment:
This can be deleted after https://github.com/patternfly/patternfly/pull/453 gets merged & released.